### ISSUE TITLE
Transparency of typing-notify.hidden not prioritized

### ISSUE THREAD

**Issue by Loctav** [cs96and/FoundryVTT-CGMP#9]
Possibly causing conflicts with other modules that alter the colors in Foundry, elements with the class `typing-notify.hidden` may still display despite receiving the `color: transparent;` attribute. Adding `!important` would fix this issue. This might also happen with modules that alter the height of elements, so maybe priorizing the `max-height` attribute similiarly may help assuring that the element stays hidden under any circumstance.

```CSS
.typing-notify.hidden {
    max-height: 0px;
    color: transparent;
}
```

to

```CSS
.typing-notify.hidden {
    max-height: 0px;
    color: transparent !important;
}
```

**Comment by cs96and** [comment]
Thanks for this.  Do you know which modules this causes an incompatibility with?  I would like to be able to test any changes before I release them.

**Comment by Loctav** [comment]
The conflicting module would be [Ernie's Modern UI](https://github.com/ernieayala/ernies-modern-layout) in my specific use-case.

**Comment by cs96and** [comment]
9c92765220063e3d869146499598b8c37135212b

**Comment by cs96and** [comment]
Fixed in 0.4.2
